### DISCUSSION TITLE
Home Assistant Container LXC Upgrade

### DISCUSSION THREAD

**Discussion by WholeSon** [tteck/Proxmox#1123] [Help Needed] [answered]
Hi, how to upgrade Home Assistant Container LXC?
When trying to run `.../homeassistant-v5.sh  script` in **LXC Console** from `# cd ~`, the following error occurs:

> Bash: line 30: pvesh: command not found ✗ No Home Assistant Installation Found!

An attempt to run the script from `cd /var/lib/docker/volumes/hass_config/_data` also resulted in this error.
What am I doing wrong? Thank's a lot.

**Comment by tteck** [comment, marked as answer]
FIXED
